File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1397.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8')
var lines = input.split('\n')

while(true){
    linha = lines.shift()
    n = parseInt(linha)
    
    if(n == 0) break
    
    a = 0
    b = 0
    for(i = 0; i < n; i++){
        ls = lines.shift().split(' ')
        v1 = parseInt(ls[0])
        v2 = parseInt(ls[1])
        
        if(v1 > v2) a++
        else if(v1 < v2) b++
        
    }
    
    console.log(a + ' ' + b)
   
}

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1397.py
while True:
    try:
        k = int(input())
        
        a = 0
        b = 0
        
        if(k == 0): break
        
        for i in range(k):
            v = [int(i) for i in input().split()]
            
            if(v[0] > v[1]): a += 1
            elif(v[0] < v[1]): b += 1
        
        print(a, b)
        
    except:
        break